front/src/BasicTable.tsx

```diff
@@ -9,42 +9,56 @@ import { useTranslation } from 'react-i18next';
 import { Button } from '@material-ui/core';
 import "./BasicTable.css";
 
-export default function BasicTable() {
-  const [offers, setOffers] = useState<any[]>([]); // TODO fix any
+
+interface OffersTypes {
+      _id: string,
+      position: string,
+      salary: string,
+      location: string,
+      company: string,
+      source: string,
+      createdAt: string,
+      url: string
+}
+
+const BasicTable: React.FC = () => {
+  const [offers, setOffers] = useState<OffersTypes[]>([]);
   const [skip, setSkip] = useState(0);
   const [loading, setLoading] = useState(true)
   const [openSnackbar, setOpenSnackbar] = useState(false)
   
   const tableRef = useRef<HTMLDivElement>(null);
+  const apiUrl = process.env.REACT_APP_API_URL
+  
+
 
   const { t, i18n } = useTranslation();
 
-  //@ts-ignore
-  const changeLanguage = (ln) => {
+  
+  const changeLanguage = (ln: string) => {
```

Comment:
It seems, that there are just two languages supported so you can make this type more strict